integrations/Lytics/browser.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable class-methods-use-this */
+// disabled these for Lytics js tag
+/* eslint-disable no-plusplus */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-sequences */
+/* eslint-disable yoda */
+/* eslint-disable prefer-spread */
+// disabling these eslint which are caused by the Lytics js tag
+
+/* eslint-disable prefer-rest-params */
+/* eslint-disable no-var */
+/* eslint-disable vars-on-top */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-shadow */
+/* eslint-disable no-void */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable one-var */
+/* eslint-disable lines-around-directive */
+/* eslint-disable strict */
+import logger from "../../utils/logUtil";
+
+class Lytics {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.stream = config.stream;
+    this.blockload = config.blockload;
+    this.loadid = config.loadid;
+    this.name = "LYTICS";
+  }
+
+  loadLyticsScript() {
+    (function () {
+      "use strict";
+      var o = window.jstag || (window.jstag = {}),
+        r = [];
+      function n(e) {
+        o[e] = function () {
+          for (var n = arguments.length, t = new Array(n), i = 0; i < n; i++)
+            t[i] = arguments[i];
+          r.push([e, t]);
+        };
+      }
+      n("send"),
+        n("mock"),
+        n("identify"),
+        n("pageView"),
+        n("unblock"),
+        n("getid"),
+        n("setid"),
+        n("loadEntity"),
+        n("getEntity"),
+        n("on"),
+        n("once"),
+        n("call"),
+        (o.loadScript = function (n, t, i) {
+          var e = document.createElement("script");
+          (e.async = !0), (e.src = n), (e.onload = t), (e.onerror = i);
+          var o = document.getElementsByTagName("script")[0],
+            r = (o && o.parentNode) || document.head || document.body,
+            c = o || r.lastChild;
+          return null != c ? r.insertBefore(e, c) : r.appendChild(e), this;
+        }),
+        (o.init = function n(t) {
+          return (
+            (this.config = t),
+            this.loadScript(t.src, function () {
+              if (o.init === n) throw new Error("Load error!");
+              // eslint-disable-next-line no-unused-expressions
+              o.init(o.config),
+                // eslint-disable-next-line func-names
+                (function () {
+                  for (var n = 0; n < r.length; n++) {
+                    var t = r[n][0],
+                      i = r[n][1];
+                    o[t].apply(o, i);
+                  }
+                  r = void 0;
+                })();
+            }),
+            this
+          );
+        });
+    })();
+
+    // Define config and initialize Lytics tracking tag.
+
+    window.jstag.init({
+      loadid: this.loadid,
+      blocked: this.blockload,
+      stream: this.stream,
+      sessecs: 1800,
+      src: `//c.lytics.io/api/tag/${this.accountId}/latest.min.js`,
```

Comment:
We recommend using an absolute URL:
```suggestion
      src: `https://c.lytics.io/api/tag/${this.accountId}/latest.min.js`,
```

Reply:
Changed it.